nightshade-webclients.yml

```diff
@@ -145,8 +119,13 @@
 
   vars:
     omero_web_config_set: "{{ omero_web_config_set_for_group | combine(omero_web_config_set_for_host) }}"
-
-    nginx_version: 1.14.2
```

Comment:
What's the reason for removing `nginx_version`?

Reply:
we did not specify for example in outreach

Reply:
This was introduced during the cyberessentials work as the Nginx verison was out of date: https://github.com/ome/prod-playbooks/pull/150/ . I think it should be brought back so we have full control over the version.

Reply:
understood